src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -67,6 +67,211 @@ def test_WINNF_FT_S_SIQ_2(self):
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_f.json')))
+    device_5 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_g.json')))
+
+    # Move device_1 inside PPA and outside 150kms range of FSS
+    device_1['installationParam']['latitude'] = 38.854
+    device_1['installationParam']['longitude'] = -97.195
+
+    # Move device_2 outside PPA and outside 150kms range of FSS
+    device_2['installationParam']['latitude'] = 40.0
+    device_2['installationParam']['longitude'] = -97.87
+
+    # Move device_3 inside PPA and inside 150kms range of FSS
+    device_3['installationParam']['latitude'] = 38.775
+    device_3['installationParam']['longitude'] = -97.291
+
+    # Move device_4 outside PPA and inside 150kms range of FSS
+    device_4['installationParam']['latitude'] = 39.0
+    device_4['installationParam']['longitude'] = -98.6
+
+    # Move device_5 outside PPA and outside 150kms range of FSS
+    device_5['installationParam']['latitude'] = 38.2
+    device_5['installationParam']['longitude'] = -99.5
+
+    # Register devices - Use same userId as in PAL record for devices part of
+    # the PPA cluster list.
+    device_1['userId'] = pal_user_id
+    device_3['userId'] = pal_user_id
+    devices = [device_1, device_2, device_3, device_4, device_5]
+    cbsd_ids = self.assertRegistered(devices)
+
+    # Add devices 1 and 3 to the cluster list of the PPA
+    ppa_record['ppaInfo']['cbsdReferenceId'] = [cbsd_ids[0], cbsd_ids[2]]
+
+    # Inject PAL and PPA records
+    self._sas_admin.InjectPalDatabaseRecord(pal_record[0])
+    zone_id = self._sas_admin.InjectZoneData({'record': ppa_record})
+    self.assertTrue(zone_id)
+
+    # Trigger daily activities and wait for it to be completed
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Spectrum Inquiries.
+    spectrum_inquiries = []
+    for i in xrange(4):
+      spectrum_inquiries.append({
+          'cbsdId': cbsd_ids[i],
+          'inquiredSpectrum': [{
+              'lowFrequency': 3550000000,
+              'highFrequency': 3700000000
+          }]
+      })
+
+    spectrum_inquiries.append({
+        'cbsdId': cbsd_ids[4],
+        'inquiredSpectrum': [{
+            'lowFrequency': 3670000000,
+            'highFrequency': 3700000000
+        }]
+    })
+
+    request = {'spectrumInquiryRequest': spectrum_inquiries}
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse']
+
+    # Check Spectrum Inquiry Response
+    self.assertEqual(len(response), 5)
+
+    # Checking response codes 0 for all inquiries. Checks for specific devices
+    # will be treated in case by case sections below.
+    for i in xrange(5):
```

Comment:
There are 5 spectrum inquiries, should be xrange(4): 0, 1, 2, 3, 4.

Reply:
xrange(4) would iterate though 0, 1, 2, 3 only.

Reply:
Oops, yes. Thanks.